AppCenterDistribute/AppCenterDistributeTests/MSDistributeCheckForUpdateTests.m

```diff
@@ -37,8 +43,12 @@ - (void)testCheckForUpdateAuthenticateWhenItHasNotAuthenticated {
   MSDistribute *distribute = [MSDistribute new];
   id distributeMock = OCMPartialMock(distribute);
   OCMStub([distributeMock canBeUsed]).andReturn(YES);
+  id parserMock = OCMClassMock([MSBasicMachOParser class]);
```

Comment:
does this need to stopMocking?

Reply:
we stopMocking for this in other tests indeed